javascript/README.md

```diff
@@ -0,0 +1,117 @@
+# Automated Testing using Appium on Kobiton
+
+## I. Setup Environment on Mac
+
+### Prerequisites
+
+#### Homebrew
+- [Homebrew](https://brew.sh/) is a package manager for the Mac.
+-  To install Homebrew, open terminal and type the following command:
+
+```bash
+  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+- This will install Homebrew on your Mac. To check the version type the following command:
+
+```bash
+  brew -v
+```
+
+#### Node and npm
+
+- To install Node via Homebrew, type the following command to install Node.
+
+```bash
+  brew install node
+```
+
+- To check if you have [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) installed, run this command in your terminal:
+
+```bash
+  node -v
+  npm -v
+```
+
+- To update your npm, type this into your terminal:
+
+```bash
+  npm install npm@latest -g
+```
+
+
+## III. Installation
+
+- Clone [this script directly from our GitHub repo](https://github.com/kobiton/samples/tree/master/javascript) and run this command in your terminal:
+  
+  ```bash
+    npm install
+  ```
+
+
+## IV. Getting started with Kobiton
```

Comment:
why not do similar with https://github.com/kobiton/samples/blob/master/java/README.md ?